tests/test_two_sided_market.py

```diff
@@ -1,16 +1,17 @@
+from math import isclose
+from unittest.mock import MagicMock, patch
 from uuid import uuid4
 
 import pendulum
 import pytest
-from d3a.models.market.blockchain_interface import NonBlockchainInterface
-from d3a_interface.dataclasses import BidOfferMatch
-from pendulum import now
-from math import isclose
-from unittest.mock import MagicMock, patch
 from d3a_interface.constants_limits import ConstSettings
+from d3a_interface.dataclasses import BidOfferMatch
 from d3a_interface.matching_algorithms import (
     PayAsBidMatchingAlgorithm, PayAsClearMatchingAlgorithm
 )
+from pendulum import now
```

Comment:
Since the imports are ordered now and I noticed, I guess one of the two should be enough like here [some lines above ](https://github.com/gridsingularity/d3a/pull/1179/commits/27956385f36abc33d073cbefbe6a98006b0a9880#diff-e443008143292bdfa0c5f6d4d871a132e7d974f5078bc4493649dbfa5b2506e4R5), whole pendulum library is imported too.

Reply:
Agree here

Reply:
Pycharm (which I used to order the imports automatically) disagrees with both of you though :D . The way the third party imports are organized are first to use the imports that use the `import` syntax (using alphanumeric order) and then to order the imports that use the `from * import * ` syntax, again in alphanumeric order.